docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,139 @@
+# Running the Airy Core Plaform in production
```

Comment:
we miss the front matter here (they configuration part of the document. You can see it in any other doc